src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/TreeView/TreeViewPage.xaml.cs

```diff
@@ -0,0 +1,1010 @@
+				GetCheckBoxStates(ContentModeTestTreeView);
+			}
+			else
+			{
+				GetCheckBoxStates(TestTreeView);
+			}
+		}
+
+		private void ToggleSelectedNodes_Click(object sender, RoutedEventArgs e)
+		{
+			if (IsInContentMode())
+			{
+				ContentModeTestTreeView.SelectionMode = TreeViewSelectionMode.Multiple;
+				var item0 = TestTreeViewItemsSource[0].Children[0];
+				var item2 = TestTreeViewItemsSource[0].Children[2];
+				var selectedItems = ContentModeTestTreeView.SelectedItems;
+				if (selectedItems.Contains(item0))
+				{
+					selectedItems.Remove(item0);
+					selectedItems.Remove(item2);
+				}
+				else
+				{
+					selectedItems.Add(item0);
+					selectedItems.Add(item2);
+				}
+			}
+			else
+			{
+				TestTreeView.SelectionMode = TreeViewSelectionMode.Multiple;
+				var node0 = TestTreeView.RootNodes[0].Children[0];
+				var node2 = TestTreeView.RootNodes[0].Children[2];
+				var selectedNodes = TestTreeView.SelectedNodes;
+				if (selectedNodes.Contains(node0))
+				{
+					selectedNodes.Remove(node0);
+					selectedNodes.Remove(node2);
+				}
+				else
+				{
+					selectedNodes.Add(node0);
+					selectedNodes.Add(node2);
+				}
+			}
+		}
+
+		private void AddInheritedTreeViewNode_Click(object sender, RoutedEventArgs e)
+		{
+			if (_visualRoot != null)
+			{
+				var node = new TreeViewNode2() { Content = "Inherited from TreeViewNode" };
+				_visualRoot.Children.Add(node);
+			}
+		}
+
+		private void ClearNodes_Click(object sender, RoutedEventArgs e)
+		{
+			try
+			{
+				if (IsInContentMode())
+				{
+					TestTreeViewItemsSource.Clear();
+				}
+				else if (_visualRoot != null)
+				{
+					_visualRoot.Children.Clear();
+					_visualRoot = null;
+					TestTreeView.RootNodes.Clear();
+				}
+			}
+			catch (Exception ex)
+			{
+				ExceptionMessage.Text = ex.ToString();
+			}
+		}
+
+		private void AddRootNode_Click(object sender, RoutedEventArgs e)
+		{
+			try
+			{
+				if (IsInContentMode())
+				{
+					var newNode = new TreeViewItemSource() { Content = "Root" + TestTreeViewItemsSource.Count };
+					TestTreeViewItemsSource.Add(newNode);
+				}
+				else
+				{
+					var newNode = new TreeViewNode() { Content = "Root" + TestTreeView.RootNodes.Count };
+					TestTreeView.RootNodes.Add(newNode);
+					if (_visualRoot == null)
+					{
+						_visualRoot = newNode;
+					}
+				}
+			}
+			catch (Exception ex)
+			{
+				ExceptionMessage.Text = ex.ToString();
+			}
+		}
+
+		private void AddExtraNodes_Click(object sender, RoutedEventArgs e)
+		{
+			for (int i = 1; i <= 50; i++)
+			{
+				TestTreeView.RootNodes.Add(new TreeViewNode() { Content = "Node " + i });
+			}
+		}
+
+		private void SelectLastRootNode_Click(object sender, RoutedEventArgs e)
+		{
+			TestTreeView.SelectedNode = TestTreeView.RootNodes[TestTreeView.RootNodes.Count - 1];
+		}
+
+		private void TreeViewLateDataInitTestPage_Click(object sender, RoutedEventArgs e)
+		{
+			//TODO:
+			//Frame.NavigateWithoutAnimation(typeof(TreeViewLateDataInitTest));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)